unitTests/ui/QuerySummaryTest.ts

```diff
@@ -170,15 +170,30 @@ export function QuerySummaryTest() {
         expect($$(test.cmp.element).find('.coveo-query-summary-no-results-string')).toBeNull();
       });
 
-      it(`when enableCancelLastAction is set to true,
-          it should display the cancel last action link on no results`, () => {
-        test = Mock.optionsComponentSetup<QuerySummary, IQuerySummaryOptions>(QuerySummary, {
-          enableCancelLastAction: true
+      describe('when enableCancelLastAction is set to true, on no results', () => {
```

Comment:
The component also be tested in the axe-core tests suite (AccessibilityTests).

Reply:
Alright, I'll add accessibility tests for it.